.github/workflows/gke.yaml

```diff
@@ -0,0 +1,102 @@
+name: GKE
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
I think these are the default types anyway https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target